src/isar_robot/robotinterface.py

```diff
@@ -143,3 +141,48 @@ def _get_battery_telemetry(self, robot_id: str) -> str:
 
     def robot_status(self) -> RobotStatus:
         return RobotStatus.Available
+
+    def _create_image(self, step: Union[TakeImage, TakeThermalImage]):
+        now: datetime = datetime.utcnow()
+        image_metadata: ImageMetadata = ImageMetadata(
+            start_time=now,
+            time_indexed_pose=TimeIndexedPose(pose=self.pose, time=now),
+            file_type="jpg",
+        )
+        image_metadata.tag_id = step.tag_id
+        image_metadata.analysis = ["test1", "test2"]
+        image_metadata.additional = step.metadata
+
+        image: Image = Image(metadata=image_metadata)
+
+        file: Path = random.choice(list(self.example_images.iterdir()))
+
+        with open(file, "rb") as f:
+            data: bytes = f.read()
+
+        image.data = data
+
+        return [image]
+
+    def _create_video(self, step: Union[TakeVideo, TakeThermalVideo]):
+        now: datetime = datetime.utcnow()
+        video_metadata: VideoMetadata = VideoMetadata(
+            start_time=now,
+            time_indexed_pose=TimeIndexedPose(pose=self.pose, time=now),
+            file_type="mp4",
+            duration=11,
+        )
+        video_metadata.tag_id = step.tag_id
+        video_metadata.analysis = ["test1", "test2"]
+        video_metadata.additional = step.metadata
+
+        video: Video = Video(metadata=video_metadata)
+
+        file: Path = random.choice(list(self.example_images.iterdir()))
```

Comment:
Will the example images work for the video type as well?

Reply:
It will work all the way to the frontend off flotilla, because it is transformed to a bytestring and passed all the way to the store

Reply:
Should we add some example videos or is it not worth it?

Reply:
I dont think it is worth it as we dont have any code for showing the uploaded videos yet. Perhaps in the future when we can visualize inspections